### PR TITLE
Bump path-to-regexp

### PR DESCRIPTION
**Pull request by FK78**
Resolves no issue.

**Overall change:** 
_Bump `path-to-regexp` to latest version._

**Code changes:**
- _Bump `path-to-regexp` from `3.2.0` to `6.1.0`._
- _Update files that use `path-to-regexp` and fix failing tests._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/InlineLink/index.jsx [hunk 1]
```diff
@@ -9,6 +8,8 @@ import fragment from '../Fragment';
 import { inlineLinkModelPropTypes } from '#models/propTypes/inlineLink';
 import { articlePath } from '../../routes/regex';
 
+const { pathToRegexp } = require('path-to-regexp');
```
Comment (12) [inline src/app/containers/InlineLink/index.jsx]: Can we not import this like previous?

Comment (FK78) [reply src/app/containers/InlineLink/index.jsx]: I've followed the method mentioned in the documentation. Using the previous import method causes test failures.

Comment (FK78) [reply src/app/containers/InlineLink/index.jsx]: Adding curly brackets around the method being imported seems to fix the failing tests.

**Review by 12 — APPROVED** [review]

**Review by tochwill — APPROVED** [review]

**Comment by PriyaKR** [comment]
@FK78 i am getting bundlesize issues when i run `npm run build`.Can you please fix this issue.

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**